source/manual/tech-lead-responsibilities.html.md

```diff
@@ -34,7 +34,7 @@ If the developer is new to GOV.UK, then it’s worth taking them through the [ov
 
 ## Managing leavers
 
-When a team member leaves GOV.UK, the team's Delivery Manager should inform Business Operations of the leaver/mover.
+When a team member leaves GOV.UK, either the person's Line Manager or their team's Delivery Manager should submit a leaver ticket to the service desk.
```

Comment:
Submitting the service desk ticket won't trigger the process for people who move to other programmes.

I actually think that this should mention submitting the [GOV.UK Bus Ops form](https://docs.google.com/forms/d/e/1FAIpQLSf5nuU8KMyutnL7K4H5URd1aIrp_BPQSrcWpALg1Ed5kU7BfA/viewform) as well. Notifying the Business Operations team triggers the process by adding a Leaver or Mover to the [TECH-2ND-LINE-VERSION]-GOV.UK-Starters-Movers-Leavers spreadsheet. Platform Support (PSR dev on a rota) should check the sheet daily and process Leaver and Mover entries or set a reminder if the last day is in the future. This ensures the request will be dealt with promptly.

Service desk ticket is the 2nd trigger for Platform Support (if the Bus Ops form was not submitted) and will ensure that all SSO is removed.
The Business Operations Leavers form also gives an option to request help with Service desk ticket “If you need help with raising the IT ticket, please provide more detail below”